packages/react/src/components/Pagination/index.tsx

```diff
@@ -0,0 +1,136 @@
+import React, { ReactElement } from 'react';
+import PropTypes from 'prop-types';
+import IconButton from '../IconButton';
+import Select from '../Select';
+
+export interface paginationProps {
+  pageSize: number;
+  totalPages: number;
+  currentPage: number;
+  totalItems: number;
+  currentItemCount: number;
+  hasPrevious: boolean;
+  hasNext: boolean;
+  mobileView?: boolean;
+  handlePageSizeChange: (pagesize: string) => void;
+  handlePageChange: (pagenumber: number) => void;
+}
+
+const Pagination = (props: paginationProps) => {
+  const mobileView = props.mobileView ? props.mobileView : false;
+
+  const previousPage = props.currentPage - 1;
+  const startIndex =
+    props.currentPage === 1 ? 1 : props.pageSize * previousPage + 1;
+  const endIndex = startIndex + props.currentItemCount - 1;
+  const current_page_index = (
+    <p className={mobileView ? 'Page_Index_Sm' : ''}>
```

Comment:
If we're following the collapsed prop, this would become a modifier according to [BEM]:(http://getbem.com/naming/)

```suggestion
    <p className={collapsed ? 'Pagination--collapsed' : ''}>
```

Following this approach ensures that css variables are specific to their components and we're not leaking "global" styles.